main.go

```diff
@@ -49,11 +51,23 @@ func main() {
 
 	ctx, cancel := context.WithCancel(context.Background())
 
+	// Minimum severity of the messages that the logger will log:
+	// info, warning, error and fatal
+	logCfg := &logger.MyLoggingConfig{
+		Level: "info",
+	}
+
+	ctx, syslogger, err := logger.SetupLogging(ctx, logCfg, "dagger-registry-2023-01-23")
```

Comment:
`dagger-registry-2023-01-23` should be an environment variable. Maybe we can use `FLY_APP_NAME`?